internal/latestrelease/test.go

```diff
@@ -0,0 +1,77 @@
+// Copyright 2022 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package latestrelease
+
+import "github.com/google/go-github/v42/github"
+
+type TestCase struct {
+	tool             string
+	currentVersion   string
+	expectNewVersion bool
+	release          *github.RepositoryRelease
+}
+
+func TestCases() []TestCase {
+	f := false
+	atlasV := "atlascli/v2.0.0"
+	mcliV := "mongocli/v2.0.0"
+	mcliOldV := "v2.0.0"
+
+	tests := []TestCase{
+		{
+			tool:             "atlascli",
+			currentVersion:   "v1.0.0",
+			expectNewVersion: true,
+			release:          &github.RepositoryRelease{TagName: &atlasV, Prerelease: &f, Draft: &f},
+		},
+		{
```

Comment:
Can we add some scenarios where the current version is in the format mongocli/v... and atlascli/v ?

Reply:
I just want to be sure that we handle that scenario as well so that we don't have to worry about tag format in the future

Reply:
Yes, I added previously but since we decided to change the makefile I removed again. Let me add it back and add the versionFromTag to the checker.